extensions/amp-story/1.0/animation.js

```diff
@@ -545,11 +546,15 @@ export class AnimationManager {
     /** @private @const */
     this.builderPromise_ = this.createAnimationBuilderPromise_();
 
+    const storeService = getStoreService(this.ampdoc_.win);
+    const animationDisabled =
```

Comment:
We can rename this var to `firstPageAnimationDisabled` to make it clear that it only impacts the first page

Reply:
Done.